### PR TITLE
add nextcord-mysql

### PR DESCRIPTION
**Pull request by Jpuf0**
this resolved issue #8 

please do let me know of any changes.

### CODE REVIEW

File: python/discord-bots/nextcord-sql/index.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import nextcord
+from nextcord.ext import commands
+from asyncmy import connect
+
+import os
+from dotenv import load_dotenv
+
+
+TESTING_GUILD_ID = 1234567890 # change to example
```
Comment (BetTD) [inline python/discord-bots/nextcord-sql/index.py:9]: Move this to .env

File: python/discord-bots/nextcord-sql/index.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+@bot.event
+async def on_ready():
+    global con
+    con = await connect(
+        host="mariadb.sparkedhost.com",
+        port=20012,
+        user="nextcord",
+        password="nextcordpass",
+        database="nextcord",
```
Comment (BetTD) [inline python/discord-bots/nextcord-sql/index.py:28]: Move this to .env as well

**Review by BetTD — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! If you could make the requested changes and also include a README file that explains how to set this up, that would be awesome.

File: python/discord-bots/nextcord-sql/index.py (hunk 2 above)
Comment (BetTD) [reply python/discord-bots/nextcord-sql/index.py:28]: Also rename the folder from `nextcord-sql` to `nextcord-mysql`, as this implementation is specific to MySQL/MariaDB and would probably require further changes for it to work with other SQL database software (e.g. PostgreSQL), for which we don't provide database servers.

File: python/discord-bots/nextcord-sql/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+nextcord
+python-dotenv
```
Comment (hwalker928) [inline python/discord-bots/nextcord-sql/requirements.txt:2]: Shouldn't asyncmy be part of this?

**Comment by Jpuf0** [comment]
should be resolved in the latest commit

**Pull request merged**